pkg/doc/vc/crypto/crypto.go

```diff
@@ -309,6 +338,63 @@ func (c *Crypto) getSigner(did, didKeyType, didPrivateKey, creator string, opts
 	}
 }
 
+// ValidateProofPurpose validates the proof purpose
+func ValidateProofPurpose(proofPurpose, method string, vdri vdriapi.Registry) error {
+	if strings.Contains(method, "did:sov") {
```

Comment:
need todo to revisit. did:sov supports authentication, for example.

Reply:
created an issue https://github.com/trustbloc/edge-service/issues/368